packages/next/pages/_document.tsx

```diff
@@ -736,6 +736,18 @@ export class NextScript extends Component<OriginProps> {
         {appScript}
         {staticMarkup ? null : this.getDynamicChunks()}
         {staticMarkup ? null : this.getScripts()}
+        {process.env.__NEXT_GRANULAR_CHUNKS ? (
+          <script
+            async
```

Comment:
IIUC `_buildmanifest` contains which asset to load when routing to a url, right?
Would this cause an increase in a scenario where the user clicks a link and the router does not know what to load?

Reply:
currently the SPA route clicks are blocked on hydration, should be available by then.